noir-projects/aztec-nr/uint-note/src/uint_note.nr

```diff
@@ -16,11 +16,11 @@ use dep::aztec::{
 #[partial_note(quote {value})]
 #[derive(Serialize)]
 pub struct UintNote {
-    // The amount of tokens in the note
-    value: U128,
     owner: AztecAddress,
     // Randomness of the note to hide its contents
     randomness: Field,
+    // The amount of tokens in the note
+    value: U128,
```

Comment:
So with this change the nullable/partial struct member has to be at the end? And do we plan to tackle this later? If yes, would create an issue for this.

Reply:
Correct. I intend to do a first pass in this PR however.